Coralogix/Sources/CoralogixRum.swift

```diff
@@ -251,8 +254,8 @@ public struct CoralogixExporterOptions {
                 mobileVitalsFPSSamplingRate: Int = 300, // minimum every 5 minute
                 instrumentations: [InstrumentationType: Bool]? = nil,
                 collectIPData: Bool = true,
+                beforeSend: (([String: Any]) -> [String: Any]?)? = nil,
```

Comment:
Just curious, why didn't you use a stricter type here to prevent the user from accessing fields like timestamp?